extends Node2D
@export var house:Sprite2D
@export var bloodRoom4:Sprite2D
@export var gabriel:Sprite2D
@export var hallWay:Sprite2D
@export var town2:Sprite2D
@export var hallWay2:Sprite2D
@export var hallWay3:Sprite2D
var part:int=0

# Called when the node enters the scene tree for the first time.
func _ready():

	#end
	if globals.npcTriggered.has("mihai") and globals.npcTriggered["mihai"]:
		pass
	#town
	elif globals.inHospital:
		text_box.queue_text("oops
		gabriel doesnt feel too well
		after that, gabriel went to the hospital:
		at the hospital:")
		self.visible=false
	
	#attic 3
	elif globals.npcTriggered.has("flapjack2") and globals.npcTriggered["flapjack2"]==true:
		text_box.queue_text("this is Gabriel
		he is
		very
		very
		very
		hungry--red")

	#attic 2
	elif globals.npcTriggered.has("cat") and globals.npcTriggered["cat"]:
		gabriel.visible=false
		text_box.queue_text("gabriel is sleeping
		gabriel is waking up")

	#attic 1
	elif globals.npcTriggered.has("rat") and  globals.npcTriggered["rat"]:
		gabriel.visible=false
		text_box.queue_text("after that, gabriel slept
		and slept and slept
		and slept for 4 years")

	#intro
	else:
		audio.playMusic()
		gabriel.visible=true
		text_box.queue_text("this is Gabriel
		intelligence: not much
		math skills: 3/10
		language in general: 1/10
		physique: very big
		note: he is very hungry
		idk lets start--red")


func _process(delta):
	#called when the textBox has just finished showing an entire conversation
	if(text_box.current_state==text_box.State.ready):
		if (globals.npcTriggered.has("mihai") and globals.npcTriggered["mihai"]):
			match part:
				0:
					audio.playMusic("dramatic")
					self.visible=false
					text_box.queue_text("and so--fast--0.04")
					part+=1
				1:
					if audio.passed(5.66):
						bloodRoom4.visible=true
						self.visible=true
						text_box.queue_text("gabriel ate a 3 month old baby--fast--0.04")
						part+=1
				2:
					if audio.passed(11.32):
						text_box.queue_text("imagine eating a 3 month old baby--fast--0.04")
						part+=1
				3:
					if audio.passed(16.98):
						bloodRoom4.visible=false
						hallWay.visible=true
						text_box.queue_text("after gabriel ate a 3 month old snack, he was kicked out of the hospital--fast--0.04")
						part+=1
				4:
					if audio.passed(22.64):
						text_box.queue_text("he then reformed and became a vegetarian--fast--0.04")
						part+=1
				5:
					if audio.passed(28.3):
						hallWay.visible=false
						town2.visible=true
						text_box.queue_text("gabriel beeng a vegetarian:--fast--0.04")
						part+=1
				6:
					if audio.passed(33.96):
						town2.visible=false
						hallWay2.visible=true
						text_box.queue_text("after 2 years gabriel returned to the hospital--fast--0.04")
						part+=1
				7:
					if audio.passed(39.62):
						hallWay2.visible=false
						hallWay3.visible=true
						text_box.queue_text("then he tripped and died--fast--0.04")
						part+=1
				8:
					if audio.passed(45.28):
						text_box.queue_text("he was in a hospital but nobody liked gabriel so they let him die--fast--0.04")
						part+=1
				9:
					if audio.passed(51):
						self.visible=false
						text_box.queue_text("rip gabriel ‚ôøü§ìüè≥Ô∏è‚Äçüåà‡∂û‡∑è--0.04")

		elif globals.inHospital:
			get_tree().change_scene_to_file("scenes/hospitalRoom1.tscn")

		#attic 3
		elif globals.npcTriggered.has("flapjack2") and globals.npcTriggered["flapjack2"]:
			if house.visible==true:
				get_tree().change_scene_to_file("scenes/alley.tscn")
			else:
				house.visible=true
				text_box.queue_text("after gabriel ate flapjack, he was kicked out from his home
				homeless?--megamind
				what adventures awaits him outside?
				how many humans will he eat throughout the game?
				:)")
				text_box.inChat=false

		#attic2
		elif globals.npcTriggered.has("cat") and globals.npcTriggered["cat"]:
			get_tree().change_scene_to_file("scenes/attic3.tscn")

		#attic 1
		elif globals.npcTriggered.has("rat") and  globals.npcTriggered["rat"]:
			get_tree().change_scene_to_file("scenes/attic2.tscn")

		#intro
		else:
			get_tree().change_scene_to_file("scenes/attic1.tscn")
			audio.playMusic("backgroundNoise0")


func textReady():
	return text_box.current_state==text_box.State.ready
#its 4:24 AM, i havent slept yet and i have to go to school in 3 hours and 20 
#sleepless?
#yesterday i fell asleep around 6.30 am and overslept. because of that i missed the math test (integrals) (wtf americans dont do integrals by default in highschool:))
"""
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚£†‚†¥‚¢ö‚°≠‚†ñ‚†í‚†ã‚†â‚†Å‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†â‚†â‚†ê‚†¶‚†•‚£í‚£¢‚†§‚£Ä‚°Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚£†‚†ã‚£¥‚£è‚°Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†à‚†ô‚†õ‚†ø‚£ó‚°§‚£Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚¢†‚¢á‚£æ‚£ø‚£ø‚£ø‚£∂‚£Ñ‚°Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†¶‚£Ñ‚°Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†ô‚†≥‚¢≠‚£í‚†§‚£Ñ‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚¢Ä‚¢è‚£æ‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£∑‚£¶‚£Ä‚°Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†à‚¢ø‚£ø‚£∂‚£§‚£Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†à‚†ô‚†∂‚£ù‚°Ü‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚£û‚£æ‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£∑‚£∑‚£∂‚£∂‚£¶‚£§‚£§‚°æ‚†ø‚†ø‚£ø‚£ø‚£ø‚£¶‚£Ñ‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†à‚¢ø‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚¢∞‚¢ª‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚°ø‚†ü‚†õ‚†ã‚†â‚†Å‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†â‚†ô‚†≥‚¢¶‚°Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚°û‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚¢Ä‚°û‚£∏‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚°ø‚†ã‚†Å‚†∞‚†∂‚£≠‚°â‚†õ‚†ì‚£Ñ‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†à‚¢¶‚°Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚¢†‚†á‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚¢∏‚†É‚£ø‚£ø‚£ø‚¢ø‚£ø‚£ø‚£ø‚£ø‚£ø‚°ø‚¢´‚†Ä‚¢†‚¢∂‚†≠‚¢≠‚£¶‚†ô‚°Æ‚†Ä‚¢π‚†Ä‚¢Ä‚£Ä‚°§‚†§‚†î‚†í‚†∂‚¢∂‚£ñ‚£í‚†í‚†¢‚¢ø‚£Ç‚†Ä‚†Ä‚†Ä‚†Ä‚£∏‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚°é‚£∞‚£ø‚£ø‚°ø‚¢∏‚£ø‚£ø‚£ø‚£ø‚†ü‚¢∞‚†É‚£Ñ‚¢∏‚°∏‚£Ñ‚°†‚¢ø‚°¥‚£É‚°¥‚†ö‚¢ö‚£©‚†§‚¢§‚£∂‚£∂‚£æ‚£ø‚°ø‚£´‚°ø‚£ø‚°∂‚†ñ‚†ª‚†¶‚¢§‚£Ä‚¢Ä‚°è‚†Ä‚†Ä‚£Ä‚£Ä‚°§‚†Ä‚†Ä‚¢∏‚†Ä
‚†Ä‚†Ä‚†Ä‚¢∞‚¢´‚£ø‚£ø‚£ø‚£á‚£®‚£ø‚£ø‚£ø‚†è‚†Ä‚£æ‚††‚†º‚†¶‚†≠‚¢Ñ‚°∏‚†û‚†â‚†Å‚†ê‚†â‚¢Ä‚£¥‚£æ‚£ø‚£ø‚£ø‚£ø‚¢ü‚£µ‚°ø‚†Å‚†à‚¢≥‚°à‚£ì‚†¶‚£à‚†ì‚¢Ω‚°≥‚¢§‚°ê‚¢õ‚†Å‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚¢Ä‚£è‚£æ‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚†É‚†Ä‚†Ä‚†π‚£Ñ‚°Ä‚£†‚†ñ‚£´‚£¥‚£ß‚£§‚£§‚£§‚£æ‚£ø‚£ø‚£ø‚£ø‚£ø‚¢ü‚£µ‚£ø‚£´‚°ê‚¢£‚°Ñ‚†Ä‚°≥‚°Ä‚†ô‚†Æ‚°≥‚£Ñ‚†ò‚†≤‚£Ω‚°ø‚£Ñ‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚°º‚£æ‚£ø‚£ø‚£ø‚£ø‚†ô‚£ü‚¢ø‚°è‚†Ä‚†Ä‚†Ä‚†Ä‚¢Ä‚°ù‚£°‚£æ‚£ø‚£ø‚£ø‚£ø‚†ø‚†ã‚†©‚£ø‚£ø‚°ø‚£ü‚£µ‚°ø‚†ã‚†Å‚†ê‚¢∑‚†Ä‚£∑‚†Ä‚†π‚£∑‚£Ñ‚†Ä‚†à‚†ª‚°∑‚°Ñ‚†ö‚¢≥‚£å‚†≥‚°Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚¢∏‚°á‚£ø‚£ø‚£ø‚£ø‚£ø‚°á‚¢ø‚£∂‚†á‚†Ä‚†Ä‚¢Ä‚°¥‚¢£‚£æ‚£ø‚£ø‚£ø‚°ø‚†ã‚†Ä‚†Ä‚¢Ä‚°¥‚¢ü‚£≠‚£æ‚°ø‚£ø‚£á‚¢†‚†Ä‚†ì‚¢†‚°á‚¢∏‚£á‚†Ä‚¢≥‚°ü‚†≥‚°Ä‚†Ä‚†ò‚£ø‚°Ñ‚†à‚¢ª‚£ß‚¢π‚†Ä‚†Ä‚†Ä
‚†Ä‚¢∏‚°∑‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£º‚£ø‚†Ä‚†Ä‚¢Ä‚°û‚£¥‚£Ø‚£ø‚£∑‚£ø‚£ø‚°ø‚†ü‚£∂‚£¶‚£º‚£æ‚£ø‚£ø‚£ø‚†Å‚£Ø‚†â‚†â‚†©‚†Ω‚†õ‚°á‚£æ‚£ø‚°Ñ‚†ò‚£ø‚†É‚¢ª‚°Ñ‚†Ä‚¢∏‚°á‚†Ä‚†Ä‚¢±‚°ø‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†ô‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£∑‚£ª‚°Ñ‚£∞‚†è‚°æ‚¢õ‚£õ‚£´‚£≠‚£Ω‚£∂‚°ü‚†â‚££‚°Ü‚¢ø‚°õ‚†ã‚£º‚°è‚£º‚°è‚†Ä‚†Ä‚†Ä‚†Ç‚£º‚°ø‚†ã‚¢∏‚°á‚†Ä‚°ü‚†á‚£º‚£ø‚°Ä‚†à‚†Ä‚†Ä‚†Ä‚¢Ä‚£∑‚£Ñ‚£Ä‚£Ä
‚†Ä‚†Ä‚†Ä‚†à‚†ª‚£ø‚£ª‚£ø‚£ø‚£ø‚£∑‚£≥‚°á‚†Ä‚†ü‚¢ø‚†â‚†â‚†â‚°ç‚£Ω‚†Ä‚†à‚¢â‚£Ø‚£ø‚¢ß‚°Ä‚£ø‚£è‚†â‚†Ä‚†Ä‚†Ä‚¢Ä‚£Ä‚†§‚†∂‚¢æ‚£æ‚†á‚¢∞‚†Å‚¢∏‚°ø‚£ø‚†á‚¢Ä‚†Ä‚†Ä‚£§‚£æ‚£ø‚†ø‚†ø‚†ã
‚†Ä‚†Ä‚†Ä‚†Ä‚°Ñ‚†à‚†õ‚†ø‚£ø‚£ø‚£ø‚£∑‚°Ω‚£æ‚†Ä‚¢∏‚†Ä‚†Ä‚†Ä‚£∑‚¢π‚°ê‚£Ñ‚£à‚£Ä‚£â‚°¥‚†õ‚†õ‚†â‚†ì‚¢Ä‚°†‚†ñ‚¢ã‚£°‚£§‚£§‚£§‚°ü‚¢†‚†è‚¢Ä‚°ü‚£∏‚°ø‚¢Ä‚†Ñ‚†Ä‚£º‚£ø‚£ø‚£ø‚°Ñ‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†à‚†ô‚†õ‚£ª‚†É‚°ê‚¢∫‚£á‚†Ä‚†Ä‚¢ø‚£ø‚£¶‚†§‚†§‚†û‚†ã‚†Ä‚†Ä‚†Ä‚¢Ä‚°¥‚¢ã‚£†‚£æ‚£ø‚£ø‚£ø‚£ø‚†ø‚°µ‚†ã‚£Ä‚£æ‚°∑‚¢ã‚£Ñ‚£º‚£Ø‚£§‚£ø‚£ø‚†á‚¢ª‚£ø‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†∏‚£Ñ‚†Ä‚†Ä‚†Ä‚£Ä‚†û‚†Å‚°¥‚†Å‚†Ä‚¢ª‚£Ü‚†Ä‚†∏‚£ø‚£ü‚¢ß‚°Ä‚†Ä‚†Ä‚£Ä‚†Ä‚£∞‚°è‚¢∞‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚†ã‚£≥‚¢ø‚£ø‚£µ‚£´‚£¥‚¢´‚£ø‚£ø‚£ø‚£ø‚£ø‚°ü‚†Ä‚£∏‚°è‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†ô‚†ø‚†≠‚†ç‚£§‚†Ä‚£∏‚†É‚†Ä‚†Ä‚¢†‚†ô‚¢∑‚¢Ñ‚°ò‚¢ø‚£¶‚£Ä‚†Ä‚†Ä‚†á‚†à‚†õ‚†ø‚£∂‚£ø‚£ø‚°ø‚†ã‚¢â‚¢¥‚†æ‚†É‚¢†‚£æ‚£ø‚£ø‚£ø‚£∏‚£ø‚£ø‚£ø‚£ø‚°ü‚†Ä‚†¥‚†ã‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚°û‚†Å‚£¥‚†á‚†Ä‚†Ä‚†Ä‚£º‚°Ä‚†à‚¢ß‚†ô‚¢≥‚£§‚°Ω‚†ø‚£∑‚£¶‚†Ä‚°Ä‚†Ä‚†Ä‚†à‚†â‚†â‚†ô‚†ã‚†Å‚†Ä‚¢†‚£ø‚†É‚¢∏‚£ø‚£ø‚£ø‚°ü‚¢∏‚£ø‚°ü‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†∏‚°Ö‚£∞‚£ª‚£§‚†Ä‚†Ä‚¢Ä‚£ß‚£á‚†Ä‚¢´‚£Ü‚†Ä‚£á‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£Ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚†á‚†Ä‚£∏‚°ø‚£ø‚£ø‚°Ü‚†à‚£ø‚†É‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†â‚¢ª‚°á‚£á‚†Ä‚†Ä‚¢∏‚¢ª‚£ø‚°Ñ‚†Ä‚†ª‚£ß‚†∏‚†ü‚†õ‚†õ‚¢ª‚£ø‚£ü‚†õ‚†â‚†Ä‚°Ø‚£≠‚£â‚†õ‚†õ‚£ø‚£ø‚°ñ‚†ã‚†Å‚¢Ä‚£ø‚£ø‚†ª‚†¶‚†å‚†É‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚£†‚£ø‚°ø‚£º‚°Ä‚†Ä‚¢∏‚£∏‚£ø‚£Ω‚¢¶‚°¥‚†É‚¢Ä‚£†‚£§‚°Ä‚†Ä‚†ë‚°ù‚¢∑‚£Ñ‚°∏‚†ß‚¢î‚£Ä‚°ª‚¢†‚£è‚†à‚°á‚†∞‚†Æ‚£õ‚¢ø‚†ô‚†ì‚†§‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚£ø‚†ü‚¢≥‚£ø‚°±‚£Ñ‚£à‚£ø‚°ø‚†ü‚£â‚£§‚£∂‚£ø‚£ø‚£ø‚£ø‚£¶‚°Ä‚°á‚†Ä‚†à‚¢∑‚£¥‚£í‚£â‚†á‚£∏‚£ø‚£≤‚£ß‚†∞‚°à‚†ø‚£ø‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚¢Ä‚£¥‚†Ä‚†Ä‚†ª‚†Ü‚†à‚†ª‚¢ß‚°π‚£π‚¢è‚£•‚£æ‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£π‚°á‚†Ä‚†Ä‚†Ä‚†π‚£ø‚°õ‚¢†‚£ø‚£ü‚£õ‚£ø‚°Ñ‚¢°‚†¥‚¢ª‚£∑‚£Ñ‚°Ä‚¢Ä‚£Ä‚£†‚£§‚£§‚°§‚°Ä‚†Ä‚†Ä
‚†Ä‚†Ä‚†Ä‚¢Ä‚£º‚£ø‚°á‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚¢†‚¢æ‚£µ‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚°Ü‚†Ä‚†Ä‚†Ä‚†Ä‚¢ª‚£ø‚†ø‚£ø‚£∂‚£ø‚£ø‚£á‚†Ä‚†æ‚†õ‚†õ‚†ß‚†ª‚†Ø‚†û‚£ã‚£•‚†Ω‚†Ü‚†â‚†ô‚†Ä
‚†Ä‚†Ä‚¢®‚£ø‚°ø‚†ü‚†Å‚†Ä‚†Ä‚†Ä‚†Ä‚£Ä‚°º‚£°‚£ø‚£ø‚£ø‚£ø‚£ø‚£ü‚£†‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚°ø‚†Ñ‚†Ä‚†Ä‚†Ä‚¢±‚†π‚£ß‚£≠‚¢ª‚°ø‚£ª‚°ü‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Å‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚¢†‚£æ‚†ü‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚¢†‚£ø‚£Ω‚°ø‚†ü‚£©‚£æ‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚°ü‚†Ä‚†Ä‚†Ä‚†Ä‚†à‚£¶‚£º‚£á‚£ø‚£∑‚°á‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚†Ä‚°º‚†Å‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚£†‚°æ‚†ã‚†Å‚¢†‚£æ‚°ø‚°ø‚†ª‚†æ‚†ø‚†ø‚¢ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚£ø‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†∞‚†õ‚£ø‚°§‚††‚°à‚†Å‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚£º‚†É‚†Ä‚†Ä‚†Ä‚£†‚†î‚†ã‚†Å‚†Ä‚†Ä‚†ê‚†ã‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚¢Ä‚£º‚£ø‚£ø‚£ø‚°ø‚£ø‚°ø‚†á‚£Ä‚£§‚°ñ‚†Ç‚†Ä‚†Ä‚†Ä‚°Ω‚†ô‚†í‚†ã‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
‚£ø‚£§‚£Ñ‚£§‚†û‚†â‚†Ä‚†Ä‚†Ä‚†Ä‚†à‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚°¥‚†ã‚†â‚†ô‚†õ‚†Å‚£†‚£ø‚£∂‚£ø‚°ü‚†â‚†Å‚†Ä‚†Ä‚†Ä‚£æ‚†Å‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä‚†Ä
"""
